notebooks/03.09-Pivot-Tables.ipynb

```diff
@@ -49,9 +49,7 @@
   {
```

Comment:
<p>This immediately gives us some insight: overall, three of every four females on board survived, while only one in five males survived!</p><p>This is useful, but we might like to go one step deeper and look at survival by both sex and, say, class. Using the vocabulary of&nbsp;<code>GroupBy</code>, we might proceed using something like this: we&nbsp;<em>group by</em>&nbsp;class and gender,&nbsp;<em>select</em>&nbsp;survival,&nbsp;<em>apply</em>&nbsp;a mean aggregate,&nbsp;<em>combine</em>&nbsp;the resulting groups, and then&nbsp;<em>unstack</em>&nbsp;the hierarchical index to reveal the hidden multidimensionality. In code:</p>
<br/>
 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>